sdks/python/apache_beam/runners/dataflow/dataflow_runner.py

```diff
@@ -566,6 +572,17 @@ def run_pipeline(self, pipeline, options):
     result.metric_results = self._metrics
     return result
 
+  def _maybe_add_unified_worker_missing_options(self, options):
+    # set default beam_fn_api and use_unified_worker experiment if
+    # 'use_runner_v2' experiment flag exists, no-op otherwise.
+    experiments = options.view_as(DebugOptions).experiments or []
+    if 'use_runner_v2' in experiments:
+      if not 'beam_fn_api' in experiments:
```

Comment:
@pabloem Do we need to add BQ option here as well?

Reply:
I think it makes sense to add it. `'use_beam_bq_sink'`.

Reply:
can we do it for batch bq source as well? I don't think the UW can support the native BQ source...

Reply:
The switch for BQ source is a little trickier. I am working on a follow up PR for that.

Reply:
okay, let's do it at least for the bq_sink then, as Pablo commented earlier.